a3p-integration/proposals/z:acceptance/package.json

```diff
@@ -79,6 +79,11 @@
     "@agoric/xsnap-lockdown": "portal:../../agoric-sdk/packages/xsnap-lockdown",
     "@agoric/zoe": "portal:../../agoric-sdk/packages/zoe",
     "@agoric/zone": "portal:../../agoric-sdk/packages/zone",
+    "@cosmjs/amino@^0.35.0-rc": "npm:^0.34.0",
```

Comment:
Why are these 34 instead of 35 like the others?

Reply:
Because this package uses `dev` client-utils published to NPM and that doesn't have 0.35 yet. Without this there's a version incompatibility.

After we publish a new client-utils to NPM this can switch to use it and drop these resolutions.